tests/core/test_bungiesearch.py

```diff
@@ -300,8 +300,7 @@ def test_optimal_queries(self):
         db_item = NoUpdatedField.objects.get(pk=1)
         src_item = NoUpdatedField.objects.search.query('match', field_title='My title')[0]
         self.assertEqual(src_item.id, db_item.id, 'Searching for the object did not return the expected object id.')
-        self.assertTrue(src_item._meta.proxy, 'Was expecting a proxy model after fetching item.')
-        self.assertEqual(src_item._meta.proxy_for_model, NoUpdatedField, 'Proxy for model of search item is not "NoUpdatedField".')
+        self.assertEqual(src_item.get_deferred_fields(), {'field_description'}, 'Was expecting description in the set of deferred fields.')
```

Comment:
Does this test fully replace the two one which were removed? It seems different (I haven't used Django 1.10, so I might be mistaken.)